javascript/bunicorn/config.yaml

```
@@ -1,6 +1,6 @@
 framework:
   github: ragokan/bunicorn
-  version: 0.1
+  version: 0.7
```

```suggestion
  version: 0.0
```

I didn't notice last time

---

Oh, so I will do this on next pull request since you merged. Thanks!

---

I thin I'll make by han'd for next update